src/Bigcommerce/Api/Client.php

```diff
@@ -57,6 +57,58 @@ class Client
      * @var string
      */
     static public $api_path;
+    static private $client_id;
+    static private $store_hash;
+    static private $auth_token;
+    static private $stores_prefix = '/stores/%s/v2';
+    static private $api_url = 'https://api.bigcommerce.com';
+    static private $login_url = 'https://login.bigcommerce.com';
+
+    /**
+     * Configure the API client with the required settings to access
+     * the API for a store.
+     *
+     * Accepts both OAuth and Basic Auth credentials
+     *
+     * @param array $settings
+     */
+    public static function configure($settings)
+    {
+        if (isset($settings['client_id'])) {
+            self::configureOAuth($settings);
+        } else {
+            self::configureBasicAuth($settings);
```

Comment:
Per the requirements, this should be removed. PM intentionally only wants to support oAuth on this latest version as we will be removing legacy API support sometime this year.